src/naive_bayes/gaussian.rs

```diff
@@ -223,6 +219,30 @@ impl<T: RealNumber, M: Matrix<T>> GaussianNB<T, M> {
     pub fn predict(&self, x: &M) -> Result<M::RowVector, Failed> {
         self.inner.predict(x)
     }
+
```

Comment:
Missing:

- [ ] class_count_ (currently not stored in the internal structures but we might need to store them for the partial_fit implementation
- [ ] epsilon_

https://github.com/scikit-learn/scikit-learn/blob/88be2abb7b7f7450dc569e0065e672a0676e4130/sklearn/naive_bayes.py#L143